### ISSUE TITLE
🐛 BUG: Language dropdown ARIA menu issue

### ISSUE THREAD

**Issue by RafidMuhymin** [dilmaheu/dilmahtea.me#246]
**Describe the bug**
The language selectors dropdown hasn't been properly implement as an ARIA menu.

**Steps to Reproduce**

1. Go to https://dilmahtea.me/ with a screen reader turned on
2. See that the language dropdown menu and its items aren't treated properly as a menu and menuitems.

**Expected behavior**
The language selectors dropdown should be properly implement as an ARIA menu.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Chrome
 - Version: 110

**Additional context**
N/A
